python/cudf_polars/cudf_polars/utils/versions.py

```diff
@@ -6,16 +6,17 @@
 # ruff: noqa: SIM300
 from __future__ import annotations
 
-from packaging.version import parse
-
-from polars import __version__
+from importlib import metadata
 
-POLARS_VERSION = parse(__version__)
+from packaging.version import parse
 
+POLARS_VERSION = parse(metadata.version("polars"))
```

Comment:
Should we continue to trust the `polars.__version__` moving forward? I think the 1.3.2 (the problematic `__version__` version?) was at least yanked on pypi

Reply:
Good catch! Done in https://github.com/rapidsai/cudf/pull/19582/commits/94657f2e92b12265689ff3fd1c9f3cd477a0880f